app/server.rb

```diff
@@ -0,0 +1,83 @@
+require 'data_mapper'
+require 'sinatra'
+require 'rack-flash'
+
+
+env = ENV['RACK_ENV'] || 'development'
+DataMapper.setup(:default, "postgres://localhost/chitter_challenge_#{env}")
+
+require_relative 'models/peep'
+require_relative 'models/user'
+require_relative './views/helpers/application'
+
+enable :sessions
+set :session_secret, 'super secret'
+use Rack::Flash
+use Rack::MethodOverride
+
+DataMapper.finalize
+DataMapper.auto_upgrade!
+
+get '/' do 
+  @peeps = Peep.all
```

Comment:
nothing here is specifying the order. It would be nice to ensure they are always in chronological order.